api-test/run-all.sh

```
@@ -0,0 +1,108 @@
+    exit 2
+  fi
+  run_bdd || echo "(bdd reported scenario failures)"
+fi
+
+if [[ ",$SURFACES," == *",e2e,"* ]]; then
+  echo "-- e2e surface (create client -> login -> token -> userinfo claims) --"
+  if [[ -z "${ESIGNET_BASE_URL:-}" ]]; then
+    echo "ESIGNET_BASE_URL is required for the e2e surface" >&2
+    exit 2
+  fi
+  run_e2e || echo "(e2e reported scenario failures)"
```

Shellcheck (0.11.0)

[info] Use run_e2e "$@" if function's $1 should mean script's $1.

(SC2119)